src/components/EmptyState/emptyState.scss

```diff
@@ -72,4 +76,4 @@
     // .subtitle a:hover {
```

Comment:
Remove this uncommented part

Reply:
This old code. someone marked it as todo. that's why left it there